### PR TITLE
updating for collectd5

### PR DESCRIPTION
**Pull request by rshade**

### CODE REVIEW

**Comment by rshade** [comment]
closes #46

File: recipes/master.rb [hunk 1]
```diff
@@ -125,3 +125,6 @@
   missing_dns_creds.map! { |cred| "rs-mysql/dns/#{cred}" }
   log "Following DNS credentials are missing #{missing_dns_creds.join(', ')}! Skipping DNS setting..."
 end
+
+node.default['rs-mysql']['collectd']['mysql']['MasterStats']='true'
```
Comment (cdwilhelm) [inline recipes/master.rb]: why did you add attributes here instead of in the attributes/default file?

File: recipes/slave.rb [hunk 2]
```diff
@@ -136,3 +136,7 @@ class Chef::Recipe
   backup false
   action :delete
 end
+
+node.default['rs-mysql']['collectd']['mysql']['SlaveStats'] = 'true'
```
Comment (cdwilhelm) [inline recipes/slave.rb]: why did you add attributes here instead of in the attributes/default file?

Comment (rshade) [inline recipes/slave.rb]: You can't do it by server type in attributes/default.rb

File: recipes/master.rb (hunk 1 above)
Comment (rshade) [inline recipes/master.rb]: You can't do it by server type in attributes/default.rb
